### PR TITLE
Cosmetic upgrades for Piano Genie demo

### PR DESCRIPTION
**Pull request by chrisdonahue**

### CODE REVIEW

**Comment by adarob** [comment]
@notwaldorf

File: piano-genie/piano_genie_bundle.js [hunk 1]
```diff
@@ -44687,43 +44687,43 @@ var categoricalDeltaTimeUserParameters = {
...
     epiano_auto_no_enc: {
-        name: 'Autoregressive',
+        name: 'Language Model',
         uri: BASE_PATH + 'model/epiano/auto_no_enc_483027',
         modelCfg: new PianoGenieModel.AutoNoEncCfg(true),
         defaultUserParameters: categoricalUserParameters
     },
     epiano_auto_no_enc_dt: {
-        name: 'Autoregressive + Delta Time',
+        name: 'Language Model w/ Timing',
```
[the remark this answers is not in the document]
Comment (notwaldorf) [reply piano-genie/piano_genie_bundle.js]: Oh yeah, that makes sense. Which model should I use for the fancy demo then?

[the remark this answers is not in the document]
Comment (notwaldorf) [reply piano-genie/piano_genie_bundle.js]: Oh yeah, that makes sense. Which model should I use for the fancy demo then?

[the remark this answers is not in the document]
Comment (chrisdonahue) [reply piano-genie/piano_genie_bundle.js]: The one selected by default (`epiano_stp_iq_auto_contour_dt` is the key for the configs dictionary).

Could we merge the minor optics improvements in this PR for now and then handle the URL reassignment once the pretty version is ready? (https://tensorflow.github.io/magenta-demos/piano-genie can point to the pretty version and maybe https://tensorflow.github.io/magenta-demos/piano-genie-comparison for the ugly one)

I'm _really_ looking forward to seeing what you have in mind.

**Comment by adarob** [comment]
@notwaldorf @chrisdonahue is this PR ready to merge?

**Comment by chrisdonahue** [comment]
Yes from my end

**Pull request merged**